### ISSUE TITLE
Provide a separate Image tile instead of using Richtext tile

### ISSUE THREAD

**Issue by MrTango** [plone/plone.app.standardtiles#104]
I realized that the old image tile way has some advantages over using the Richtext tile, especially for pre defined content layouts, where you want to have images as place holders and want to allow the user just to change the image but not the size or the location of the image. With the richtext tile we can prevent the moving but can not have pre defined fixed sizes. That would be really help full. I can imagine that it should be possible to create an Image tile, which uses the upload pattern to upload an image to the current context, either next to a document or inside it, if it is a folderish type.
The tile should allow to set the meta data like the description and so on as well es pre define a scale for the image. The mosiac content layout  should be able to disallow changing the scale.

**Comment by datakurre** [comment]
I believe that the code for separate image tiles is still there. The issue with it was that it required duplicating much of the scaling code in plone.namedfile and that code should be manually kept in sync with plone.namedfile changes. Also image tile blobs are not versioned yet.

**Comment by agitator** [comment]
@MrTango it would be handy to have an image tile and extend it with a target link (and optional override of title and description) to be used as a teaser tile.

**Comment by datakurre** [comment]
We are using in-house tiles with similar features, but with uuid-linked image instead of image saved with with tile. It was not as convenient as images saved in tile data for just images, but has been working pretty well with leadimage-behaviored content.

**Comment by mauritsvanrees** [comment]
> I believe that the code for separate image tiles is still there.

True. But @jensens is removing it for [Plone 6](https://github.com/plone/plone.app.standardtiles/pull/127).

> The issue with it was that it required duplicating much of the scaling code in plone.namedfile and that code should be manually kept in sync with plone.namedfile changes.

True. But I have recently made it easier to customise or override parts of `plone.namedfile`. Latest `plone.app.tiles` alpha uses this. The remaining customisations are really only about getting data from the tile instead of the content item. So I believe this reason does not count anymore.

> Also image tile blobs are not versioned yet.

I have not checked, but this is probably still true.
